Engine/Fuego/Windows/WindowWin.cpp

```diff
@@ -172,9 +172,15 @@ WindowWin::WindowWin(const WindowProps& props, EventQueue& eventQueue)
                             nullptr);
     FU_CORE_ASSERT(m_Hwnd, "[AppWindow] hasn't been initialized!");
     hwndMap.emplace(m_Hwnd, this);
+
+    _context = new OpenGLContext(&m_Hwnd);
+    FU_CORE_ASSERT(_context->Init(),
+                   "[GraphicsContext] hasn't been initialized!");
+
```

Comment:
That line is too short to  move to another line

Reply:
clang-format is at work